.github/workflows/rspec_tests.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: RSpec test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  rspec_tests:
+    name: ${{ matrix.cfg.os }}(ruby ${{ matrix.cfg.ruby }})
+    strategy:
+      matrix:
+        cfg:
+          - {os: ubuntu-18.04, ruby: '2.5'}
+          - {os: ubuntu-18.04, ruby: '2.7'}
+          - {os: ubuntu-18.04, ruby: '3.0'}
+          - {os: ubuntu-18.04, ruby: 'jruby-9.2.17.0'}
+          - {os: windows-2019, ruby: '2.5'}
+          - {os: windows-2019, ruby: '2.7'}
+          - {os: windows-2019, ruby: '3.0'}
+          - {os: windows-2019, ruby: 'jruby-9.2.17.0'}
+
+    runs-on: ${{ matrix.cfg.os }}
+    steps:
+      - name: Checkout current PR
+        uses: actions/checkout@v2
+
+      - name: Install ruby version ${{ matrix.cfg.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.cfg.ruby }}
+
+      - name: Update rubygems and install gems
+        run: |
+          gem update --system --silent --no-document
+          bundle config set without packaging development
```

Comment:
I don't completely understand why, but you'll want to remove `development` from this list. Otherwise the [rake "development_dependency"](https://github.com/puppetlabs/semantic_puppet/blob/90782698bb00ca92eb480da04d69142f8e45c339/semantic_puppet.gemspec#L26) won't be installed.